bittensor/metagraph.py

```diff
@@ -913,4 +1182,5 @@ def load_from_path(self, dir_path: str) -> "metagraph":
         return self
 
 
-metagraph = Metagraph
+print("USE_TORCH", use_torch())
```

Comment:
why don't use `btlogging` instead of `print`?

Reply:
This shouldn't be there at all. It was for testing. Good catch!